diff --git a/widget/pulse.lua b/widget/pulse.lua
index fb7244a..a75dd10 100644
--- a/widget/pulse.lua
+++ b/widget/pulse.lua
@@ -187,6 +187,9 @@ function pulse.new(args, style)
 	widg._sink = args.sink
 	table.insert(pulse.widgets, widg)

+	-- some unification between standard monitor gauge and special audio gauge
+	widg.set_mute = widg.set_mute or widg.set_alert
+
 	-- Set tooltip
 	--------------------------------------------------------------------------------
 	widg._tooltip = tooltip({ objects = { widg } }, style.tooltip)